desktop/core/src/desktop/templates/hue.mako

```diff
@@ -503,6 +525,17 @@ ${ commonHeaderFooterComponents.footer(messages) }
   ${ _('Hue and the Hue logo are trademarks of Cloudera, Inc.') }
   <b>Query. Explore. Repeat.</b>
 </div>
+<script>
+  document.addEventListener("click", function(event){
```

Comment:
this would push every click to google analytics and wouldn't differentiate between parent click or element click. we should try to capture based on the type of elements.